boards/xtensa/esp32/esp32_defconfig

```diff
@@ -32,3 +32,8 @@ CONFIG_I2C_1=y
 CONFIG_CLOCK_CONTROL=y
 
 CONFIG_BOOTLOADER_ESP_IDF=y
+CONFIG_HEAP_MEM_POOL_SIZE=98304
+CONFIG_MINIMAL_LIBC_MALLOC_ARENA_SIZE=4096
+CONFIG_POSIX_API=y
```

Comment:
esp-idf uses `gettimeofday()` internally. This is the only reason to use `POSIX_API` for now.
We plan to update and remove this dependency.

Reply:
Okay. Could you add a comment above `CONFIG_POSIX_API=y` describing it?

Reply:
Is `gettimeofday()` used by WiFi driver or by other things as well? Because I wonder why do you enable it in the commit with WiFi. If it is only needed by WiFi, but not other features, then there should be `select POSIX_API` in wifi driver Kconfig instead.